sql/core/src/test/scala/org/apache/spark/sql/CollationStringExpressionsSuite.scala

```diff
@@ -70,6 +74,152 @@ class CollationStringExpressionsSuite extends QueryTest with SharedSparkSession
     })
   }
 
+  test("INSTR check result on non-explicit default collation") {
```

Comment:
this doesn't seem to test any collation

Reply:
Shouldn't we always make sure that previous behaviour was preserved for backwards compatibility? But I do agree that separate test is maybe not necessary, but could be merged with the following test.

Reply:
My point is, we should already have tests like this, which evaluate the expressions without any collation feature.